pkg/reconciler/delivery/table_test.go

```diff
@@ -36,6 +36,11 @@ import (
 	. "knative.dev/serving/pkg/testing/v1"
 )
 
+var (
+	// key is test Key, value is a string representing the event triggered during reconciliation, if any
+	fakeEvents = make(map[string]string)
```

Comment:
This is a global variable. Test variable should be created each test run, that way there is no accidental test bleed.